server/graphql/programs/src/mutations/vaccination/insert.rs

```diff
@@ -94,6 +100,7 @@ fn map_error(error: ServiceError) -> Result<InsertVaccinationResponse> {
     let graphql_error = match error {
         ServiceError::VaccinationAlreadyExists
         | ServiceError::ClinicianDoesNotExist
+        | ServiceError::FacilityNameDoesNotExist
```

Comment:
Small nit, but could we call this "FacilityDoesNotExist"? It makes it seem like we're checking the name field, but we're not, we're checking that the ID exists as a Facility name (right?) (I was initially confused as I wondered why we'd check the name for a free text field). (Plus it's slightly more consistent with the "ClinicianDoesNotExist" right above it)